.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@sebbss , this file should not be pushed to github

Reply:
i removed it, pushed it by mistake